tests/src/Phansible/Roles/PhpTest.php

```diff
@@ -77,7 +79,7 @@ public function testShouldGetInitialValues()
     public function testShouldNotTransformValues()
     {
         $values = [
-            'packages' => ['php5-cli', 'php5-intl']
+            'packages' => ['php5-cli', 'php5-intl'],
```

Comment:
```suggestion
            'packages' => ['php5-cli', 'php5-intl']
```

Reply:
I genuinely don't know why this is happening, those commas aren't on my local working copy.